packages/ui-extensions/buildTargetDts.ts

```diff
@@ -72,14 +72,20 @@ function createInitialTargetDefinition({
   const directory = join(buildPath, `ts/surfaces/${surface}/targets`);
   const targetPath = join(directory, fileName);
 
-  const template = `import type {ExtensionTargets} from '../extension-targets';
+  let template = `import type {ExtensionTargets} from '../extension-targets';
```

Comment:
Does this work? In the future I think each surface should have its own globals and we could always import it but for now we can tie it to customer-account only.

```ts
  const template = `import type {ExtensionTargets} from '../extension-targets';
type Target = ExtensionTargets[${name}];
export type Api = Target['api'];
export type Output = Target['output'];
${
  surface === 'customer-account'
    ? `import '../globals';\n`
    : ''
}
\n`;
```

Reply:
Updated!